templates/validator/validator.html

```diff
@@ -164,9 +166,12 @@
             </ul>
             <div class="tab-content h-100" id="myTabContent">
               <div id="charts" class="tab-pane fade w-100 h-100 show active" role="tabpanel" aria-labelledby="charts-tab">
-                <div class="btn-group border rounded mt-2 ml-2 charts-btn-group" role="group" aria-label="Charts buttons group">
-                  <button type="button" class="btn btn-link btn-sm border-right income-chart-btn nav-link">Income</button>
-                  <button type="button" class="btn btn-link btn-sm proposed-chart-btn nav-link">Proposals</button>
+                <div class="d-flex justify-content-between flex-nowrap px-2 pt-2">
+                  <div class="btn-group border rounded charts-btn-group" role="group" aria-label="Charts buttons group">
+                    <button type="button" class="btn btn-link btn-sm border-right income-chart-btn nav-link">Income</button>
+                    <button type="button" class="btn btn-link btn-sm proposed-chart-btn nav-link">Proposals</button>
+                  </div>
+                  <button type="button" id="load-income-btn" class="btn btn-link btn-sm border rounded nav-link">Load More</button>
```

Comment:
The button style and placement does not match the design drat attached to our BIDS. Have you discussed this with @Buttaa ? If not, please do so?

Reply:
Fixed, ptal.